patches/tools-win-update_idl.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tools/win/update_idl.py b/tools/win/update_idl.py
+index b8f7aa8ff246de468b4596ef04bf8f9d3070733b..4c660343efaa8cae84d25a0f8d1a7e6575c653e4 100644
+--- a/tools/win/update_idl.py
++++ b/tools/win/update_idl.py
+@@ -48,6 +48,7 @@ class IDLUpdater:
+                     f'is_debug=true\n'
+                     f'blink_symbol_level=0\n'
+                     f'v8_symbol_level=0\n').format()
++        contents += 'translate_genders=false\n'; open(gn_args_path, 'wt', newline='').write('translate_genders=false\n') if not os.path.exists(gn_args_path) else None
```

Comment:
Ideally I'd prefer this to be a chromium_src override for `IDLUpdated::_gen_gn_args()`, but since this is a script to run manually, a patch is fine.

why not do it this way though?
```diff
                     f'is_debug=true\n'
+                    f'translate_genders=false\n'
                     f'blink_symbol_level=0\n'
```